### DISCUSSION TITLE
Any example setting cookies in crossOrigin parameter?

### DISCUSSION THREAD

**Discussion by matteomontipo** [lhz516/react-h5-audio-player#168] [Q&A]
hi,
is there any example setting up cookies inside the crossOrigin parameter?

I've read it should be set to 'use-credentials' in order to use a cookie, but I'm not sure how the player accepts these data,
something like:   crossOrigin={'use-credentials': { myCookies } } or crossOrigin={`use-credentials: ${myCookies}`} ? idk

**Comment by lhz516** [comment]
The cookie will be automatically sent with the request

**Reply by matteomontipo** [reply]
I'm sorry if it wasn't clear from my question, I'd need to send a specific cookie, how should I set it inside crossOrigin?

**Reply by lhz516** [reply]
I think all cookies will be sent with the request, you can't specific one of them.
